docs/sftt/sftt-setup-backend.md

```diff
@@ -0,0 +1,32 @@
+# Setting up the Backend
+This is a comprehensive overview of the steps required and everything needed to run the database locally. 
+
+### Downloads
+1. Clone the [Github repo](https://github.com/Code-4-Community/speak-for-the-trees-backend)
+2. Download [PostgreSQL](https://www.postgresql.org/download/) (Make sure it comes with pgAdmin4)
+3. Download [Java JDK 8](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
+4. Download [Java JRE 8](https://www.oracle.com/java/technologies/javase-jre8-downloads.html)
```

Comment:
When I downloaded and installed JDK 8 on windows I didn't have to do this with the installer, but maybe you need to on different platforms?